src/api/exchange/DepositApiImpl.ts

```diff
@@ -155,25 +155,27 @@ export class DepositApiImpl implements DepositApi {
 
   /********************************    private methods   ********************************/
 
-  private async _getContract(): Promise<EpochTokenLocker> {
+  protected async _getNetworkId(): Promise<number> {
     let networkId = getNetworkIdFromWeb3(this._web3)
 
-    if (networkId === null) {
-      networkId = await this._web3.eth.net.getId()
-    }
+    return networkId === null ? this._web3.eth.net.getId() : networkId
```

Comment:
Don't you need to await `this._web3.eth.net.getId()`?

Reply:
no, `async` function return promises, so you don't return an `await` you just `return`